sdk/src/main/scala/com/horizen/account/mempool/exception/MempoolException.scala

```diff
@@ -0,0 +1,9 @@
+package com.horizen.account.mempool.exception
+import sparkz.util.ModifierId
+
+import java.math.BigInteger
+
+class MempoolException(message: String) extends Exception(message)
```

Comment:
Do we want to move `TxOversizedException` here as well?

Reply:
I agree. Done